.mci.yml

```diff
@@ -355,8 +357,10 @@ functions:
                   set -o xtrace
 
                   if [ "Windows_NT" == "$OS" ]; then
-                      CRUD_TESTS_PATH=$(cygpath -m $CRUD_TESTS_PATH)
+                      CRUD_LEGACY_TESTS_PATH=$(cygpath -m $CRUD_LEGACY_TESTS_PATH)
+                      CRUD_UNIFIED_TESTS_PATH=$(cygpath -m $CRUD_UNIFIED_TESTS_PATH)
                       ENCRYPTION_TESTS_PATH=$(cygpath -m $ENCRYPTION_TESTS_PATH)
+                      UNIFIED_FORMAT_TESTS_PATH=$(cygpath -m $UNIFIED_FORMAT_TESTS_PATH)
```

Comment:
Windows tests were complaining about being unable to find files at `UNIFIED_FORMAT_TESTS_PATH` so I added this in; unsure how they were passing before...

Reply:
Ahh clever, thanks for looking into this... Yep, looks like I was just overwriting `UNIFIED_FORMAT_TESTS_PATH` by accident. Removed the path variables here, but kept the `CTEST_OUTPUT_ON_FAILURE` env variables below.

Reply:
Looks like `ENCRYPTION_TESTS_PATH` is still needed, too. Not quite sure why, as the same overwriting is not happening in `test/CMakeLists.txt`.